src/pageObject.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return 'Click on the ' + this.header.logo;
+  }
 };
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super(url + slug);
+  }
+
+  get commentButton() {
+    return '[Publish comment] button';
+  }
 
+  clickOnCommentButton() {
+    return 'Click on the ' + this.commentButton;
```

Comment:
Consider using template literals for string concatenation to improve readability: `return `Click on the ${this.commentButton}`;`